### ISSUE TITLE
EVENT_AFTER_PAYMENT_TRANSACTION isn't triggered

### ISSUE THREAD

**Issue by chrismou** [craftcms/commerce#1817]
**Description**
In the Payments service there seems to be a fully documented event called EVENT_AFTER_PAYMENT_TRANSACTION which I was hoping to use to listen for an authorize request being completed, but it doesn't look like it's actually triggered anywhere. Am I missing something?

https://github.com/craftcms/commerce/blob/develop/src/services/Payments.php#L42
https://craftcms.com/docs/commerce/3.x/events.html#afterpaymenttransaction

**To Reproduce**
```
Event::on(
            Payments::class,
            Payments::EVENT_AFTER_PAYMENT_TRANSACTION,
            function(TransactionEvent $event) {
                // @var Transaction $transaction
                $transaction = $event->transaction;
            }
        );
```

**Expected behavior**
What did you expect to happen?

**Additional info**
- Craft version: 3.5.14
- Commerce version: 3.2.8.1
- PHP version: 7.3
- Database driver & version: MySQL

**Comment by chrismou** [comment]
I've put in a PR where it looks like it should be, let me know if there's anything else you need to get it merged

**Comment by lukeholder** [comment]
Thanks! fixed in #1823 (Had to make a new branch to make some adjustments).

Fix will be in the next release.
